aspnetcore/blazor/common/samples/3.x/BlazorSample/Components/HTTPRequestTester.razor

```diff
@@ -0,0 +1,178 @@
+@using System.Net
```

Comment:
Update sample code to preview6 and apply changes suggested for the snippets above.